File: backend/src/main/java/kr/krproject02/Krproject02Application.java
package kr.krproject02;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;

@SpringBootApplication(exclude = {SecurityAutoConfiguration.class})
public class Krproject02Application {

    public static void main(String[] args) {
        SpringApplication.run(Krproject02Application.class, args);
    }

}

File: backend/src/main/kotlin/kr/krproject02/Krproject02Application.kt
package kr.krproject02

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = [SecurityAutoConfiguration::class])
class Krproject02Application

fun main(args: Array<String>) {
    runApplication<Krproject02Application>(*args)
}
